workers/unity/Packages/com.improbable.gdk.mobile/Editor/LaunchMenu.cs

```diff
@@ -0,0 +1,60 @@
+using System.Collections;
+using System.IO;
+using System.Text;
+using Improbable.Gdk.Tools;
+using UnityEditor;
+using UnityEngine;
+
+namespace Improbable.Gdk.Mobile
+{
+    public class LaunchMenu
+    {
+        [MenuItem("SpatialOS/Mobile/Launch on Device", false)]
```

Comment:
add priority as we did for all the other menu items

Reply:
What menu priority should this have? The priorities list for the "other menu items" is current in the tools package, and it is internal.

Reply:
yeah, some are in tools, some are in build system. Basically wherever you want it to show up